Gemfile.lock

```diff
@@ -109,47 +100,6 @@ GEM
       specinfra (~> 2.10)
       syslog-logger (~> 1.6)
       uuidtools (~> 2.1.5)
-    chef (13.4.24-universal-mingw32)
```

Comment:
that seems a bit weird, probably need to `rake dependencies:update_gemfile_lock`